tidy3d/components/simulation.py

```diff
@@ -49,6 +49,9 @@
 MAX_CELLS_TIMES_STEPS = 1e17
 MAX_MONITOR_DATA_SIZE_BYTES = 10e9
 
+# number of grid cells at which we star warning about slow Simulation.epsilon()
+NUM_CELLS_WARN_EPSILON = 274_000_000
```

Comment:
This is about 650 cells in each direction - just curious as to how this number was chosen? Assuming a double precision epsilon array where each entry is 8 bytes, I guess this number leads to about 2.2GB RAM - or 4.4GB for complex numbers. Was this the main consideration?

Reply:
This was just chosen empirically through playing around with the tests and picking the number where sim.epsilon took a second or so to run.

Reply:
Ok, thanks.